client/llb/source.go

```diff
@@ -452,6 +452,45 @@ func Differ(t DiffType, required bool) LocalOption {
 	})
 }
 
+func OCILayout(contentStoreIDAndHash string, opts ...OCILayoutOption) State {
```

Comment:
`contentStoreIDAndHash` break it into 2 variables. Lets avoid microformats here. By Hash here you mean Digest?

Reply:
> By Hash here you mean Digest

Yes.

I actually had it as 2 vars in an interim stage, but look a few lines down, and it is merged together to create:

```go
	source := NewSource("oci-layout://"+contentStoreIDAndHash, attrs, gi.Constraints)
```

so what do we get here by splitting it and then merging it?

Reply:
It basically already _is_ a microformat natively; I am not sure what we can do to change that.

Reply:
Yes, it's a bit weird as your input is already a string that is a join of two fields but better to keep the Go API clean. The caller might not parse it from as string and may have them separately.

Reply:
Sure, I don't mind.